CHANGELOG.md

```diff
@@ -21,7 +21,10 @@ Adding a requirement of a major version of a dependency is breaking a contract.
 Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian/virtual-users/compare/release-3.15.1...master
+[Unreleased]: https://github.com/atlassian/virtual-users/compare/release-3.15.2...master
+
+## [3.15.2] - 2023-10-12
```

Comment:
NIt: https://packages.atlassian.com/maven-central/com/atlassian/performance/tools/virtual-users/3.15.2/ says `11-Oct-2023`. Maybe we are taking into account timzeones?